<?xml version="1.0"?>
<launch>
    <node pkg="follow_path_creator" type="test_path_creator" name="test_path_creator" output="screen">
        <param name="world_frame_id" value="map"/>
        <param name="resolution" value="0.05"/>
        <param name="course_length" value="7.0"/>
        <param name="init_x" value="2.0"/><!-- -4.957 -->
        <param name="init_y" value="0.3"/><!-- 1.886 -->
        <!-- <param name="init_theta" value="0.75"/>&#60;&#33;&#45;&#45; -0.355 &#45;&#45;&#62; -->
        <param name="init_theta" value="0.0"/><!-- -0.355 -->
        <param name="A1" value="0.5"/>
        <param name="omega1" value="85"/>
        <!-- <param name="omega1" value="85"/> -->
        <param name="delta1" value="220"/>
        <param name="A2" value="0.0"/>
        <param name="omega2" value="0"/>
        <param name="delta2" value="0"/>
        <param name="A3" value="0.0"/>
        <param name="omega3" value="0"/>
        <param name="delta3" value="0"/>
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find follow_path_creator)/config/rviz/test_path_creator.rviz"/>
</launch>
